### PR TITLE
Added test 'changes Supplier'

### PR DESCRIPTION
**Pull request by ivodimitrov**

### CODE REVIEW

**Comment by ivodimitrov** [comment]
[Item details - Supplier & Currency](https://trello.com/c/6MD4rtOJ/113-item-details-supplier-currency)

**Comment by ivodimitrov** [comment]
Refactored tests and replaced `screens` from `constants` to `ios`

File: spec/ios/direct_client/unintegrated_costs_item_details_spec.rb [hunk 1]
```diff
@@ -63,39 +63,25 @@
   it 'changes Supplier on item details screen', :tms => '51514' do
     on(ItemDetailsScreen).add_details_text
     on(ItemDetailsScreen).supplier_text
-    on(ItemSupplierScreen).checkmark_button?
-    on(ItemSupplierScreen).search_for_supplier @supplier_name
+    on(ItemSupplierScreen).check_mark_button?
+    on(SearchTextField).search_for_text @supplier_name
```
Comment (ivarbanovv) [inline spec/ios/direct_client/unintegrated_costs_item_details_spec.rb:67]: search_for_supplier was better

Comment (ivarbanovv) [reply spec/ios/direct_client/unintegrated_costs_item_details_spec.rb:67]: but I see why the change

**Pull request merged**